### PR TITLE
apply now page

### PR DESCRIPTION
**Pull request by vishaal93**
some restructuring + blank page for apply now form

### CODE REVIEW

File: static/css/styles.css [hunk 1]
```diff
@@ -6,6 +6,11 @@
   background: #EAC452;
 }
 
+.header-link {
```
Comment (Nikhil22) [inline static/css/styles.css]: You can use the tachyon classes no-underline for text-decoration none, and white for color: white

File: src/components/NavModal.vue [hunk 2]
```diff
@@ -8,7 +8,7 @@
         <div class="pa3 f3 pointer" @click="configObj.onAboutClick()">About</div>
         <div class="pa3 f3 pointer" @click="configObj.onStoryClick()">Story</div>
         <div class="pa3 f3 pointer" @click="configObj.onContactClick()">Contact</div>
-        <div class="pa3 f3 pointer" @click="configObj.onJoinClick()">Join</div>
+        <div class="pa3 f3 pointer" @click="configObj.onApplyNowClick()">Join</div>
```
Comment (Nikhil22) [inline src/components/NavModal.vue]: If we are going with Apply Now then the text should also be changed to Apply Now

File: static/css/styles.css (hunk 1 above)
Comment (vishaal93) [reply static/css/styles.css]: oh shit thats lit LOL

File: src/components/Apply/ApplyNow.vue [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<template>
+  <div class="tc">
+
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'ApplyNow',
+  data () {
```
Comment (Nikhil22) [inline src/components/Apply/ApplyNow.vue]: this is not needed

File: static/css/styles.css (hunk 1 above)
Comment (Nikhil22) [reply static/css/styles.css]: Yeah with Tachyons we have to literally write no custom CSS

File: src/components/NavModal.vue (hunk 2 above)
Comment (vishaal93) [reply src/components/NavModal.vue]: Did the changes in the UI.  Not sure where this join is but will make the change

screen shot 2018-01-31 at 11 01 12 pm

Comment (Nikhil22) [reply src/components/NavModal.vue]: It's for Mobile

File: src/components/Apply/ApplyNow.vue (hunk 3 above)
Comment (vishaal93) [reply src/components/Apply/ApplyNow.vue]: yeah, kept it in there for when I add the child components so I don't have to google the format

**Pull request merged**